### DISCUSSION TITLE
Telemetry on LoRa APRS packets

### DISCUSSION THREAD

**Discussion by richonguzman** [richonguzman/LoRa_APRS_iGate#122] [Announcements]
Hi @all

this is met to discuss the requested modes for Telemetry to see how we could add this into igate Firmware

right now there are Wx formatted packets and also telemetry packets

Wx data can be passed over telemetry format but the benefit of Wx formatted packet is that only one packet is sended with:
gps + wx data

instead Telemetry needs another packet to be sended with the formatted info, but enables us to send any data that we need

**Comment by zaldyrs** [comment]
Hi, its been a while i didnt open mu LoRa. I found new feature with the web flasher, its awesome. I am able to manage it for my tbeam axp2102 v1.2. Then I try to flash for my heltec lora 32 v2, it showed 'Error. Failed to download manifest'. I am not sure what happened, i use same usb cable although. Maybe you can advice if i missed some steps.
Cheers, thanks. 73

**Reply by richonguzman** [reply]
Where (which link) are you trying?

**Reply by zaldyrs** [reply]
Hi, i tried this link : https://richonguzman.github.io/lora-igate-web-flasher/installer.html, and select heltec lora 32 v2. Still not working. I tried to upload other firmware from TinyGs, it works. Thanks to advice me.

**Reply by richonguzman** [reply]
But your problem is that the board is not working or that it does not send telemetry?

**Reply by zaldyrs** [reply]
When i ran using web flasher it didnt download, mentioned ‘failed to download manifest’ if i select heltec_lora32-v32 option on the dropdown menu.

**Comment by richonguzman** [comment]
please test now as I hope to have fixed the "wrong" manifest download issue

**Comment by GedaZ** [comment]
I need help, colleagues. I've installed the latest software, version 2024.08.14, with a connected BME680 sensor. It detects the sensor, but only displays temperature, humidity, and pressure on the screen. It doesn't show air quality (gases). And it sends only these three parameters (temperature, humidity, and pressure) to APRS-IS as APRS weather. Below my APRS description, it adds the word GAS: (value)Kohms. It's as if there's no code for displaying Air Quality (gas) weather. My board is TTGO LoRa32 V2.1 + BME680.

**Reply by richonguzman** [reply]
Hi,


I did not find any way to convert this value in Ohms into something usefull

do you have any chart to do so?

**Reply by GedaZ** [reply]
Converting is one thing, but why does it show me the GAS value in my APRS description? 
I only know that Air Quality (AQ) is composed of 75% Gas and 25% Humidity.

**Reply by richonguzman** [reply]
this is the value the sensor/boards... without any processing

**Reply by GedaZ** [reply]
I know it's a value from the board, but why doesn't it display in the weather data like temperature, but only in my description (and it doesn't display the GAS value on the LoRa display)?

**Reply by richonguzman** [reply]
aprs has no format to display it as the regular weather aprs packet

**Reply by richonguzman** [reply]
check APRS101.pdf file :

http://www.aprs.org/doc/APRS101.PDF

to see about it

**Reply by GedaZ** [reply]
OK thanks. Maybe I'll try with BME280 so as not to disturb GAS :)

**Comment by wb2osz** [comment]
APRS101.PDF is missing all the corrections, new features , and clarifications since 2000.

All of those updates have been merged into the version here:  https://github.com/wb2osz/aprsspec  

73,
John WB2OSZ

**Reply by richonguzman** [reply]
thanks! lets hope soon to have yours as a valid and updated APRS120.pdf file ;)
